frontend/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@gc-digital-talent",
+  "name": "gc-digital-talent",
```

Comment:
Did you mean to check in this change?  What is it doing?

Reply:
No, I did not mean to check this in. I'm still unsure as to what was happening but `npm` was complaining about the @ in the package name which is odd to say the least. I removed it temporarily to get deps installed and it got caught by an indiscriminate `git add .`

Resolved by [revert package name change](https://github.com/GCTC-NTGC/gc-digital-talent/pull/2413/commits/bd97baae2dd5e24087e632c053815999d29de6ed)